fire/api/firedb/hent.py

```diff
@@ -198,7 +198,7 @@ def hent_observationer_fra_opstillingspunkt(
             filtre.append(observationsklasse.observationstidspunkt <= tid_til)
 
         if kun_aktive:
-            filtre.append(observationsklasse._registreringtil == None)
+            filtre.append(observationsklasse._registreringtil is None)
```

Comment:
Den her ændring er tvivlsom. SQLAlchemy overloader `==` som en del af ORM-magien. Lidt mere her: https://stackoverflow.com/a/18998106

Der kan evt smides en `# noqa` kommentar efter for at undgå brok fra linters